ksql-streams/src/main/java/io/confluent/ksql/execution/streams/StreamAggregateBuilder.java

```diff
@@ -187,6 +191,19 @@ private StreamAggregateBuilder() {
             resultSchema
         );
 
+    reduced = reduced.transformValues(
+        () -> new KsTransformer<>(new WindowBoundsPopulator()),
+        Named.as(StreamsUtil.buildOpName(
+            AggregateBuilderUtils.windowSelectContext(aggregate)
+        ) + "2")
+    );
```

Comment:
Note: the dodgily named `Aggregate-Aggregate-WindowSelect2` node here will go with the next PR.  It's temporary only.

Reply:
REmoved in https://github.com/confluentinc/ksql/pull/4459